hack/build.sh

```diff
@@ -9,6 +9,22 @@ declare -a VERSIONS=(2.4)
 OS=$1
 VERSION=$2
 
+function squash {	
+	base=$(grep ^FROM Dockerfile | awk '{print $2}')
+	# install the docker layer squashing tool
+	easy_install --user pip
```

Comment:
# ~/.local/bin/pip install --user docker-scripts=0.3.2
Exception:
Traceback (most recent call last):
  File "/root/.local/lib/python2.7/site-packages/pip-6.1.1-py2.7.egg/pip/basecommand.py", line 246, in main
    status = self.run(options, args)
  File "/root/.local/lib/python2.7/site-packages/pip-6.1.1-py2.7.egg/pip/commands/install.py", line 308, in run
    name, None, isolated=options.isolated_mode,
  File "/root/.local/lib/python2.7/site-packages/pip-6.1.1-py2.7.egg/pip/req/req_install.py", line 220, in from_line
    isolated=isolated)
  File "/root/.local/lib/python2.7/site-packages/pip-6.1.1-py2.7.egg/pip/req/req_install.py", line 79, in __init__
    req = pkg_resources.Requirement.parse(req)
  File "/root/.local/lib/python2.7/site-packages/pip-6.1.1-py2.7.egg/pip/_vendor/pkg_resources/__init__.py", line 2960, in parse
    reqs = list(parse_requirements(s))
  File "/root/.local/lib/python2.7/site-packages/pip-6.1.1-py2.7.egg/pip/_vendor/pkg_resources/__init__.py", line 2904, in parse_requirements
    "version spec")
  File "/root/.local/lib/python2.7/site-packages/pip-6.1.1-py2.7.egg/pip/_vendor/pkg_resources/__init__.py", line 2869, in scan_list
    raise ValueError(msg, line, "at", line[p:])
ValueError: ('Expected version spec in', 'docker-scripts=0.3.2', 'at', '=0.3.2')